config/default.py

```diff
@@ -16,6 +16,7 @@
 SECURITY_PASSWORD_HASH = "bcrypt"
 SECURITY_PASSWORD_SALT = "abcde1234509876zyxwvu22"
 SECURITY_LOGIN_USER_TEMPLATE = "index.html"
+SECURITY_REGISTER_USER_TEMPLATE = "index.html"
```

Comment:
Does the modal pop up if you click, "I don't have an account." in the sidenav? Can we do that instead of redirecting to a new page?